test/detect/livescript/default.txt

```diff
@@ -36,3 +36,21 @@ const t2 =
   where id1 is id2]
   |> sort-by \id
   |> JSON.stringify
+
+~function add x, y
+  @result = x + y
+
+class A
+  (num) ->
+    @x = num
+  property: 1
+  method: (y) ->
+    @x + @property + y
+
+a = new A 3
+a.x        #=> 3
```

Comment:
Feels a little contrived, is this the only way you could make the detects happy?

Reply:
Yeah unfortunately that's what this is about. I just found a likely looking snippet on the LiveScript site, and copied it in here to boost the relevance.